functions/Get-DbaAgentJob.ps1

```diff
@@ -52,6 +60,8 @@ FUNCTION Get-DbaAgentJob {
 		$SqlCredential,
 		[object[]]$Job,
 		[object[]]$ExcludeJob,
+		[ValidateSet('True', 'False')]
+		[string]$Enabled,
```

Comment:
I know have the time I am pulling jobs from a server I am only concerned with the enabled ones initially.